ansible/roles_ocp_workloads/ocp4_workload_bookbag_user/defaults/main.yaml

```diff
@@ -0,0 +1,23 @@
+---
+become_override: false
+ocp_username: opentlc-mgr
+silent: false
+
+# Project in which to deploy the bookbag deployment
+ocp4_workload_bookbag_user_project: "bookbag-{{ guid }}"
+ocp4_workload_bookbag_user_project_display: "Bookbag for user {{ ocp_username }}"
+
+# Name of bookbag deployment
+ocp4_workload_bookbag_user_deployment_name: bookbag
+
+# Bookbag Image
+ocp4_workload_bookbag_user_image_namespace: "bookbags"
+ocp4_workload_bookbag_user_image_name: image-registry.openshift-image-registry.svc:5000/bookbag/bookbag
+ocp4_workload_bookbag_user_image_tag: latest
+
+# Bookbag configuration
+ocp4_workload_bookbag_user_console_deploy: true
+ocp4_workload_bookbag_user_console_image: quay.io/openshift/origin-console:4.5
```

Comment:
The console image should match the OpenShift cluster version... can we make the default reference the cluster version?

Reply:
Changed to "override". Extracting image from deployed console if override not specified.